pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1878-dep-upgrade-SNAPSHOT</version>
```

```suggestion
    <version>1.0-SNAPSHOT</version>
```

The tests have passed in our CI. I am now reverting this change and will be closing the other (test) pull request. Thank you very much! :)